CONDUCT.md

```diff
@@ -0,0 +1,131 @@
+# Conjur Code of Conduct
+
+Conjur exists to make DevOps easier, by making it easy for teams to communicate
+about and follow their security plans.
+
+Conjur is a piece of technology, but more importantly, it is a community.
+
+We believe that our mission is best served in an environment that is friendly,
+safe, and accepting; free from intimidation or harassment.
+
+Towards this end, certain behaviors and practices will not be tolerated.
+
+## tl;dr
+
+* Be respectful.
+* We're here to help: <abuse@conjur.org>
+* Abusive behavior is never tolerated.
+* Violations of this code may result in swift and permanent expulsion from the
+  Conjur community.
+
+## Scope
+
+We expect all members of the Conjur community, including paid and unpaid agents,
+administrators, users, and customers of CyberArk, to abide by this Code of
+Conduct at all times in all Conjur community venues, online and in person, and
+in one-on-one communications pertaining to Conjur affairs.
+
+This policy covers the usage of the Conjur hosted eval server, as well as the
+Conjur website, Conjur related events, and any other services offered by or on
+behalf of CyberArk. (collectively, the "Service"). It also applies to behavior
+in the context of the Conjur Open Source project communities, including but not
+limited to public GitHub repositories, Slack and IRC channels, social media,
+mailing lists, and public events.
+
+This Code of Conduct is in addition to, and does not in any way nullify or
+invalidate, any other terms or conditions related to use of the Service.
+
+The definitions of various subjective terms such as "discriminatory", "hateful",
+or "confusing" will be decided at the sole discretion of the Conjur abuse team.
```

Comment:
this is the only place that "confusing" is referenced in the document, and I'm not sure how it's relevant. if we may ban people for being confusing, I think that's worth expounding on. if including "confusing" on this list isn't meant to imply that someone could be banned for being "confusing", then I think this line could use some rewording.